guides/README.md

```diff
@@ -269,6 +342,20 @@ Meilisearch **response** :
 ```
 
 
+### Afterword
+
+
+In MeiliSearch we have three concepts on which we build our search engine. If you did not go to those pages, they give an essential insight.
```

Comment:
I don't understand what you wanted to mean with the "If you did not go to those pages". Is it more "If you haven't read these pages yet, ..." ?

Reply:
Yes great suggestion